ignite/src/main/java/org/hibernate/ogm/datastore/ignite/IgniteDialect.java

```diff
@@ -621,12 +621,8 @@ else if ( !queryParameters.getPositionalParameters().isEmpty() ) {
 							backendQuery.getSingleEntityMetadataInformationOrNull().getEntityKeyMetadata()
 						);
 		}
-		else if ( backendQuery.getQuery().isHasScalar() ) {
+		else  {
 			throw new NotYetImplementedException();
-//			return new IgniteProjectionResultCursor( result, backendQuery.getQuery().getCustomQueryReturns(), queryParameters.getRowSelection() );
-		}
-		else {
-			throw new UnsupportedOperationException( "Not implemented yet" );
```

Comment:
@schernolyas You must be more precise when you write an error message,  if somebody receive this error he will wonder what's not yet implemented

Reply:
Oops...I confused the deletion with the remaining code